scopes/dependencies/pnpm/pnpm.package-manager.ts

```diff
@@ -42,6 +42,7 @@ import { pnpmPruneModules } from './pnpm-prune-modules';
 import type { RebuildFn } from './lynx';
 import { type DependenciesGraph, type DirectDependency } from '@teambit/legacy/dist/scope/models/version';
 import * as dp from '@pnpm/dependency-path';
+import { type GetDependenciesGraphOptions } from '@teambit/dependency-resolver/package-manager';
```

Comment:
this should be exported (and imported) from the index file without the `/package-manager`